### ISSUE TITLE
Update to use CachedStaticFilesStorage

### ISSUE THREAD

**Issue by dracos** [mysociety/pombola#706]
"I have not changed to using CachedStaticFilesStorage, because that involves making sure that the assets referred to in the CSS files exist, otherwise it dies - and currently the asset files are all in core, but you only have one country app in INSTALLED_APPS and so it can't find static assets stored in other country apps. The country-based CSS files should be in their respective apps, importing SASS files from core as necessary. Once that is done, moving to cached file storage and removing the generation number should be mostly straightforward - the only issue will be the files loaded by loader.js that won't be rewritten. We could extend CachedStaticFilesStorage and add a pattern that matches the static_url() used there for rewrite, perhaps."

**Comment by paullenz** [comment]
Assume this relates to caching - can we just use cloud flare.com or similar?

**Comment by dracos** [comment]
It doesn't relate to outside caching, no. Superseded by use of django-pipeline (and moving of asset files at that time) mid 2014.
